### DISCUSSION TITLE
Welcome to opencrx-documentation Discussions!

### DISCUSSION THREAD

**Discussion by opencrx** [opencrx/opencrx-documentation#15] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by javac9** [comment]
This is great! Kudos to the Opencrx Team!

**Reply by nyemiks** [reply]
Thanks. This is good for the project.

**Comment by fschon** [comment]
+1
